mbuild/box.py

```diff
@@ -2,7 +2,7 @@
 
 import numpy as np
 
-__all__ = ['_BoxArray']
+__all__ = ['Box', '_BoxArray']
```

Comment:
Kind of a nit, but `_BoxArray` shouldn't be included in `__all__` since it isn't supposed to be used outside of this module. 

For example, this shouldn't be allowed:
```python3
>>> from mbuild.box import *
>>> _BoxArray
<class 'mbuild.box._BoxArray'>
```